components/omega/doc/devGuide/QuickStart.md

```diff
@@ -198,36 +198,36 @@ ctest log file located at `$BUILD_DIR/Testing/Temporary/LastTest.log`.
 +==============+==============+===================================================================================================================================================================+
-| chicoma-cpu  | gnu          | /usr/projects/e3sm/polaris/chicoma-cpu/spack/dev_polaris_0_3_0_gnu_mpich/var/spack/environments/dev_polaris_0_3_0_gnu_mpich/.spack-env/view                       |
+| chicoma-cpu  | gnu          | /usr/projects/e3sm/polaris/chicoma-cpu/spack/dev_polaris_0_4_0_gnu_mpich/var/spack/environments/dev_polaris_0_4_0_gnu_mpich/.spack-env/view                       |
 +--------------+--------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| chrysalis    | intel        | /lcrc/soft/climate/polaris/chrysalis/spack/dev_polaris_0_3_0_intel_openmpi/var/spack/environments/dev_polaris_0_3_0_intel_openmpi/.spack-env/view                 |
+| chrysalis    | intel        | /lcrc/soft/climate/polaris/chrysalis/spack/dev_polaris_0_4_0_intel_openmpi/var/spack/environments/dev_polaris_0_4_0_intel_openmpi/.spack-env/view                 |
 |              +--------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-|              | gnu          | /lcrc/soft/climate/polaris/chrysalis/spack/dev_polaris_0_3_0_gnu_openmpi/var/spack/environments/dev_polaris_0_3_0_gnu_openmpi/.spack-env/view                     |
+|              | gnu          | /lcrc/soft/climate/polaris/chrysalis/spack/dev_polaris_0_4_0_gnu_openmpi/var/spack/environments/dev_polaris_0_4_0_gnu_openmpi/.spack-env/view                     |
 +--------------+--------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| frontier     | gnu          | /ccs/proj/cli115/software/polaris/frontier/spack/dev_polaris_0_3_0_gnu_mpich/var/spack/environments/dev_polaris_0_3_0_gnu_mpich/.spack-env/view                   |
+| frontier     | gnu          | /ccs/proj/cli115/software/polaris/frontier/spack/dev_polaris_0_4_0_gnu_mpich/var/spack/environments/dev_polaris_0_4_0_gnu_mpich/.spack-env/view                   |
 |              +--------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-|              | gnugpu       | /ccs/proj/cli115/software/polaris/frontier/spack/dev_polaris_0_3_0_gnugpu_mpich/var/spack/environments/dev_polaris_0_3_0_gnugpu_mpich/.spack-env/view             |
+|              | gnugpu       | /ccs/proj/cli115/software/polaris/frontier/spack/dev_polaris_0_4_0_gnugpu_mpich/var/spack/environments/dev_polaris_0_4_0_gnugpu_mpich/.spack-env/view             |
```

Comment:
This build (as well as `crayclanggpu`) will require an update once we have migrated to new modules.  This path will not change when that happens, I will just update the build.  I wouldn't normally do this but I do not believe the current build is likely to work so a rebuild seems prudent.